### ISSUE TITLE
Grunt Build and BuildProd using Relative Paths

### ISSUE THREAD

**Issue by robgrimball** [balderdashy/sails#3323]
It appears that performing a grunt build or grunt buildProd sets up all assets linked with relative paths.  This also appears to be something that was done purposefully, although I am uncertain why.  Uncertain because it also appears that when you run sails lift or sails lift --prod, those paths are all linked with absolute paths.  

This was an issue for us of course, since we were expecting the paths to be absolute, and I am still puzzled as to why the path would be relative or how that would ever work.  Our fix was fairly simple for this case, we simply changed linkAssetsBuild and linkAssetsBuildProd to always use absolute paths (by changing for instance sails-linker:prodJsRelative to sails-linker:prodJs).

I guess I am wondering if this is a bug, or if it is a feature that is used for some other use case.  If a feature, I would be interested in what that entails.

**Comment by sailsbot** [comment]
Thanks for posting, @robgrimball.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
